pkg/ui/src/util/format.ts

```diff
@@ -75,7 +75,7 @@ export function BytesWithPrecision(bytes: number, precision: number): string {
  */
 export function Percentage(numerator: number, denominator: number): string {
   if (denominator === 0) {
-    return "100%";
+    return "??%";
```

Comment:
I'm inclined to use something subtler visually, perhaps "-" or "--%".  We don't want to draw attention to the lack of value, just acknowledge it.

@josueeee?

Reply:
Good idea; done.